### ISSUE TITLE
Game.InputBox inserts duplicated characters with new keyboard handling

### ISSUE THREAD

**Issue by ericoporto** [adventuregamestudio/ags#1653]
**Describe the bug**
When using Game.InputBox, the input gets twice the characters, when `Use old-style keyboard handling` is set to **false**.

**AGS Version**
Happens in 3.6.0, and current master https://github.com/adventuregamestudio/ags/commit/0372082c3f64adeb1645ea049ad70b3bfe3b36d8

But doesn't happen in 3.5.1

**Game**
[GameInputBox_3.5.1.zip](https://github.com/adventuregamestudio/ags/files/8628307/GameInputBox_3.5.1.zip)
This game project is made in 3.5.1, run with the 3.6.0 engine and it will work fine.

But rebuild it with 3.6.0 and set "Use old-style keyboard handling" in General Settings to `false`...
[GameInputBox_3.6.0.zip](https://github.com/adventuregamestudio/ags/files/8628319/GameInputBox_3.6.0.zip)

**To Reproduce**
1. Make a room with the following code
```AGS Script
function room_AfterFadeIn()
{
  String result = Game.InputBox("Double characters");
  Display(result);
}
```
2. Begin the game and press keys on the input box
3. Notice that key presses get duplicated.

**Expected behavior**
A key press should show only one character

**Desktop:**
 - OS: Windows
 - Version 10

**NOTE:** Since I don't think InputBox is used often nowadays and this doesn't affect previous games, I would say this is a low priority issue.

**Comment by ivan-mogilko** [comment]
This happens also to a regular TextBox. But I think there's some other condition, like ASCII mode has to be set in General Settings.

**Comment by ivan-mogilko** [comment]
> This happens also to a regular TextBox. But I think there's some other condition, like ASCII mode has to be set in General Settings.

Oh, nevermind, apparently this is already fixed. So InputBox may have similar bug in a separate code.

**Comment by ivan-mogilko** [comment]
Pushed a fix: adec08a02664c0ab35ab358ca255ca6f2e3801b1

**Comment by ericoporto** [comment]
This fixed it! Thanks!
